index.html

```diff
@@ -644,6 +659,28 @@ <h2>The <code>failure_fraction</code> member</h2>
         <em>all</em> <a>failed</a> <a>network requests</a> for this origin.
         </p>
       </section>
+
+      <section>
+        <h2>The <code>request_headers</code> member</h2>
+
+        <p>
+        The OPTIONAL <dfn><code>request_headers</code></dfn> member defines the
+        list of <a data-lt="policy request headers">request headers</a> that
+        will be included in <a>network error reports</a> about this
+        <a>origin</a>.  If present, its value MUST be a list of strings.
+        </p>
+      </section>
+
+      <section>
+        <h2>The <code>response_headers</code> member</h2>
+
+        <p>
+        The OPTIONAL <dfn><code>response_headers</code></dfn> member defines the
+        list of <a data-lt="policy response headers">response headers</a> that

```

Comment:
Is it worth clarifying that you intend to append _both_ header field name *and* value in these reports? Ditto for both request and response.

Reply:
Done